modules/identities/internal/mappable/identity.go

```diff
@@ -59,3 +61,55 @@ func NewIdentity(id types.ID, immutableProperties types.Properties, mutablePrope
 		HasMutables:   baseTraits.HasMutables{Properties: mutableProperties},
 	}
 }
+
+func (identity identity) IsProvisioned(address sdkTypes.AccAddress) bool {
```

Comment:
## Code Styling
### Interface Implementation
* Code should be ordered as:
```go
type struct foo{}

var _ Foo=(*foo)(nil)

func (foo foo) bar1(){}
func (foo foo) bar1(){}
func (foo foo) bar2(){}

func baz(){}

// Baz
func Baz(){}
```
* The methods implemented should be in the same order as that defined in the interface definition.